app/settings/categories/categories-edit.html

```diff
@@ -76,9 +77,10 @@ <h1 class="mode-context-title" ng-show="!category.id" translate>app.add_category
                             type="radio"
                             name="parent_label"
                             id="parent_label-nothing"
-                            value="null"
+                            ng-value="null"
                             ng-checked="!category.parent"
                             ng-model="category.parent_id"
+                            ng-click="deselectParent()"
```

Comment:
This should really be using checklist-model to avoid the messiness of so many ng- directive trying to work together